sdk/storage/azure-storage-common/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## 12.0.0 (2021-06-08)
 
+### Other Changes and Improvements
+
+- Fixed a filename encoding issue.
```

Comment:
Isn't this a bug fix?

Reply:
I wouldn't say it's a bug. We hadn't supported unicode filename before this fix was checked in. It's like an improvement or enhancement.